src/test/kotlin/com/cultureamp/pg/PgTestConfig.kt

```diff
@@ -0,0 +1,13 @@
+package com.cultureamp.pg
+
+import org.jetbrains.exposed.sql.Database
+
+// To use this, you need to input your database details
+object PgTestConfig {
+    val testerName = ""
+    val jdbcUrl = "jdbc:postgresql://localhost:5432/${testerName}"
+    val driver =  "org.postgresql.Driver"
+    val user = testerName
+
+    val db = if(jdbcUrl != "" && user != "") Database.connect(url = jdbcUrl, driver = driver, user = user) else null
+}
```

Comment:
Just to make sure I understand, is the idea that an individual can edit this file in local development in order to test against PG, but in the build we're using H2 still?

Reply:
Yes, the same as before.